sashimi-webapp/src/components/editor-viewer/Navbar.vue

```diff
@@ -1,38 +1,44 @@
 <template>
   <div class="section group navbar">
     <div class="col button-back vertical-align-child">
-        <button class="navbar-buttons">
-          <img src="../../assets/buttons/back-button.png" class="button-img" alt="back">
-        </button>
+        <a href="/" class="vertical-align-child">
```

Comment:
Vue router should be used to route between the views instead. 
Otherwise, the browser will initialize a new http request to the web application when `href="/"` is used.